### PR TITLE
1086/partner cypress tests

### PR DESCRIPTION
**Pull request by YazeedLoonat**
- Closes [#issue](https://github.com/CityOfDetroit/bloom/issues/1086)

## Description
Updates the cypress tests for partner's in detroit

## How Can This Be Tested/Reviewed?
run a re-seed and run the cypress tests

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [x] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by emilyjablonski** [comment]
@YazeedLoonat Looks like the tests are still failing in CI, could we get them green pre-review?

**Comment by YazeedLoonat** [comment]
@seanmalbert @emilyjablonski this is fully set up and working properly, lemme know what you think

File: .travis.yml [hunk 1]
```diff
@@ -1,94 +1,69 @@
...
-    PGUSER=travis
-    TEST_DATABASE_URL=postgres://localhost:5433/bloom_test
-    REDIS_TLS_URL=redis://127.0.0.1:6379/0
-    NEW_RELIC_ENABLED=false
-    NEW_RELIC_LOG_ENABLED=false
+  global:
+  - PGPORT=5433 PGUSER=travis DATABASE_URL=postgres://localhost:5433/bloom_detroit
+    TEST_DATABASE_URL=postgres://localhost:5433/bloom_test REDIS_TLS_URL=redis://127.0.0.1:6379/0
+    NEW_RELIC_ENABLED=false NEW_RELIC_LOG_ENABLED=false
+  - secure: iVBAxWMlvmBTcSq5qQhASJauYaL9IhA45e4J2WCL9ybf07aAqbpTSPG7O/V+ZBQ7mHRx9Io6U5SZVIYOhKoCwjIdSQeyzikeppRFJZwgC1T1cPqeIIWLRNjSVL+kmyPSjwZxNZTJsferlQYX/wLHteX3Rq6/kBvb2WKjwNu/mF66y7xdER0JpLgMQ3V6bGz1Ji16GVY7LYsbOdwCI1pGH/Q3swCXnLZlBixRtCEy+yFQzaz60WZgjLlDyM60+5TcvtcZFPZuO5LNyJd29iSqbR2t0EsvBTxs9j9GC8a4ySLcPK9qMOme2q3O8b/5CzC1CmjUW6XmXsd5LXyHNV+JTx/DotkTzk5k972z8awxzG+VDiTEO04zGqUQfhRi2IAgGYz0zFsRwO5pKLEnbQz5LOCC12zy1s+2d2OE9OoW/xOEQjVz/4QJr8Z49UWze1p5Y3rRUx4+++Di0A8vPufYHAkmUmxZP/Qguj0ngAn+GnFk/82UpilW/HWwkDUit6pz+xVvDTJ9eCkJYwfZkrAw3qFYm73PkYkI1HR0puMKX0V/rGY3r18nZzPpzUltAZQblSZKexqpcZKGu8avcNxHUXtKS6BY462XxCG4HUuwyrRH9N7e5Tos13rI4X18jCnWdKQ0jB4szpFOfqxqzuvyQ7UMwRYt0qantXhkDVo7qvo=
```
Comment (emilyjablonski) [inline .travis.yml:66]: Can we remove these?

**Review by emilyjablonski — APPROVED** [review]
One quick question about the yml secrets, and then I think good to go!
We do need to add some additional fields to the listing, but I think we should do that in a followup :)
The fields we aren't touching:
* Region
* Programs
* A11y features
* Due date
* Waitlist
* Marketing status coming soon with season
* Paper app uploads
* Custom online app URL
* Custom pick up and drop off apps
* Verification

File: .travis.yml (hunk 1 above)
Comment (seanmalbert) [reply .travis.yml:66]: @YazeedLoonat , once you address this, we're good to merge. I also create a follow-up issue per @emilyjablonski comment. https://github.com/CityOfDetroit/bloom/issues/1308

**Pull request merged**